### PR TITLE
Enable new "help" section and fix  a few things

### PR DESCRIPTION
**Pull request by RichieEscarez**
Add website elements to support the new "docs help" section:

- Added text for Menu links
- Replaced the content of CONTRIBUTING and DEVELOPMENT with links to this new "help" section.
- Add the option to run a "production" version of a local build
 
OTHER DEPENDENT PRS:
This is the knative/website PR, and is 1 of 3:
  - https://github.com/knative/docs/pull/3437
  - https://github.com/knative/community/pull/557

https://github.com/knative/docs/issues/2960

### CODE REVIEW

**Comment by RichieEscarez** [comment]
This PR preview: https://deploy-preview-288--knative.netlify.app/

See the related docs changes here: https://606e66140a771100079874e5--knative.netlify.app/help/contributor/publishing/

**Comment by RichieEscarez** [comment]
The content has been merged: https://github.com/knative/docs/pull/3437

This PR can go in now to show/expose both the Template and Docs Help links in the right nav menu:

https://deploy-preview-288--knative.netlify.app/docs

image

https://deploy-preview-288--knative.netlify.app/blog

image

File: assets/scss/main.scss [hunk 1]
```diff
@@ -60,3 +60,28 @@ footer {
...
+        padding: .5em;
+        background: lightgray;
+    }
+    td {
+        border: 1px solid lightgray;
+        margin-left: .5em;
+        margin-right: .5em;
+        padding: 5px;
+    }
+}
```
Comment (evankanderson) [inline assets/scss/main.scss:87]: This is updating styling for _all_ tables. I'm assuming that is intended?

**Review by evankanderson — COMMENTED** [review]
One comment about the table styling, but:

/lgtm
/hold

File: assets/scss/main.scss (hunk 1 above)
Comment (evankanderson) [reply assets/scss/main.scss:87]: In particular, compare: https://deploy-preview-288--knative.netlify.app/docs/eventing/sources/#knative-sources vs https://knative.dev/docs/eventing/sources/#knative-sources

Can we revert this one file?

Comment (RichieEscarez) [reply assets/scss/main.scss:87]: Sorry, just got out of a block of meetings. Yes. Will revert now.

Thanks for pointing this out. It looks like the table CSS was not working correctly for me locally. Initially, tables were rendering blank for me, thus adding borders. But sadly I dropped the ball here and failed to circled back around to testing this in the Netlify Preview builds.

**Comment by RichieEscarez** [comment]
Another LGTM, please (the bot removed it from the last commit)

**Pull request merged**